### ISSUE TITLE
Import doesn't work with WooCommerce

### ISSUE THREAD

**Issue by arkadiyasuratov** [wpbullet/wp-menu-import-export-cli#3]
If my menu contains a product category than during an import I get a warning `'The submenu item "' . $menu_item['title'] . '" does not have any data.'`

When I change this line:
https://github.com/wpbullet/wp-menu-import-export-cli/blob/f59bba6fea655610f2954bb91366ace93869994c/includes/trait-import.php#L170

to:

`$term = get_term_by( 'id', $menu_item['term'], $menu_item['taxonomy'] );` than my import is successful. 

What's the reason behind the original line, why a taxonomy term is pulled by a name and not id?

Thanks a bunch for this great package!

**Comment by wpbullet** [comment]
Thanks for reporting the bug @arkadywtf we will see if we can get this sorted out soon! Glad you are enjoying the package otherwise :)

**Comment by roelmagdaleno** [comment]
Hi @arkadywtf 

The code was originally this way from the repo we forked from because the owner used `$term->name` when exporting the JSON file.

That is something we changed in our version, we started to use `$term->term_id` to be more specific when exporting the JSON file, but we forgot to change the `name` value when importing (the code line you mentioned). :sweat_smile:

We're going to fix that. Thanks for noticing it and we're glad you're enjoying.
